packages/monitoring-system/data-processor/test/unit/data-processors/cloud-tasks-data-processor.spec.ts

```diff
@@ -40,10 +40,22 @@ describe('Cloud Tasks Data Processor', () => {
   let MockFirestoreData3: FirestoreData;
 
   function resetMockData() {
```

Comment:
I changed `loadFixture`'s `copy` param to be `false` by default instead of `true` so had to adjust parameters here.